nipoppy/workflows/dataset_init.py

```diff
@@ -37,20 +41,38 @@ def __init__(
             dry_run=dry_run,
         )
         self.fname_readme = "README.md"
+        self.bids_source = bids_source
 
     def run_main(self):
-        """Create dataset directory structure."""
+        """Create dataset directory structure.
+
+        Create directories and add a readme in each.
+        Copy boutiques descriptors and invocations.
+        Copy default config files.
+
+        If the BIDS source dataset is requested, it is copied.
+        """
         # dataset must not already exist
         if self.dpath_root.exists():
             raise FileExistsError("Dataset directory already exists")
 
         # create directories
         for dpath in self.layout.dpaths:
-            self.mkdir(dpath)
+
+            # If a bids_source is passed it means datalad is installed.
+            if self.bids_source is not None and dpath.stem == "bids":
+                self.logger.info(
+                    f"Installing datalad BIDS raw dataset from {self.bids_source}."
+                )
+                copytree(self.bids_source, str(dpath))
+            else:
+                self.mkdir(dpath)
 
         for dpath, description in self.layout.dpath_descriptions:
             fpath_readme = dpath / self.fname_readme
-            if description is not None and not self.dry_run:
+            if (description is not None and not self.dry_run) and not (
+                self.bids_source is not None and dpath.stem == "bids"
+            ):
```

Comment:
the BIDS dataset most likely already contains a README

Reply:
Maybe we could still add the Nipoppy README if `dpath / "README"` doesn't exist? Not sure how strict BIDS is with the name/extension of the README file though so it might not be a very simple check

Reply:
If we are adding a README to a BIDS dataset then I'd rather go with the default one from BIDS: https://github.com/bids-standard/bids-starter-kit/blob/main/templates/README.MD

Reply:
I am okay with copying the default BIDS README to the `bids` subdirectory in all Nipoppy datasets

Reply:
do you prefer to have it in the repo or to fetch it from wherever it lives?

Reply:
I would fetch it from wherever it lives, and we could have a test to check that the remote file does exists

Reply:
We should also make sure that `nipoppy init` won't crash if the default BIDS README cannot be fetched for whatever reason

Reply:
yeah that may also add "request" as a dependency, unless we want to stick to urllib

Reply:
I'm okay with a `requests` dependency, especially since we will (someday) move the pipeline files (descriptors, invocations, etc.) somewhere else and may want a relatively easy way for users to add a pipeline to their dataset

Reply:
have not forgotten about this: just forgot to push
will do as soon as I can restart my computer (currently without a charger for it)